src/storage/txn/commands/commit.rs

```diff
@@ -59,11 +59,10 @@ impl<S: Snapshot, L: LockManager> WriteCommand<S, L> for Commit {
 
         let rows = self.keys.len();
         // Pessimistic txn needs key_hashes to wake up waiters
-        let mut released_locks = ReleasedLocks::new(self.lock_ts, self.commit_ts);
+        let mut released_locks = ReleasedLocks::new();
         for k in self.keys {
             released_locks.push(commit(&mut txn, &mut reader, k, self.commit_ts)?);
         }
-        released_locks.wake_up(context.lock_mgr);
```

Comment:
It seems we no longer need `lock_mgr` in `WriteContext`.

Reply:
Yes. Let's do these kind of fixups in other PRs.